Quaver.Shared/QuaverGame.cs

```diff
@@ -185,6 +185,31 @@ public string Version
         /// </summary>
         private bool WindowActiveInPreviousFrame { get; set; }
 
+        private event EventHandler WindowFocusChanged;
+
+        private bool windowUnfocused;
+
+        /// <summary>
+        ///     Whether the window is unfocused.
+        /// </summary>
+        private bool WindowUnfocused
+        {
+            get => windowUnfocused;
+            set
+            {
+                if (value != windowUnfocused)
+                {
+                    windowUnfocused = value;
+                    WindowFocusChanged?.Invoke(this, null);
+                }
+            }
+        }
```

Comment:
This event should be implemented at [Framework level](https://github.com/Quaver/Wobble/blob/master/Wobble/WobbleGame.cs), so that all games that use Wobble can take advantage of it.